tests/std/tests/P0528R3_cmpxchg_pad/test.cpp

```diff
@@ -53,8 +70,35 @@ struct alignas(4) X4 {
     void set(char v) {

```

Comment:
Nitpick: Some, but not all, of the `set()` functions are being changed to take their parameters by `const` value.